README.md

```diff
@@ -2,9 +2,13 @@
 De documentatie is nog volop in ontwikkeling
 ```
 
-# Kennisbank
+## OSPO-NL Kennisbank
 
-Deze documentatie is gepubliceerd op [ospo-nl.github.io/kennisbank](https://ospo-nl.github.io/kennisbank/).
+De OSPO-NL kennisbank is een verzameling van kennis en best practices voor het opzetten en uitvoeren van OSPO's (of OSPOs) bij organisaties in Nederland.
+
+De OSPO-NL kennisbank is gepubliceerd op [ospo-nl.github.io/kennisbank](https://ospo-nl.github.io/kennisbank/).
+
+![image](https://user-images.githubusercontent.com/3628277/222470157-8fffb097-7d34-49b3-8283-da70a20ed5a4.png)
```

Comment:
Een screenshot is goed ... maar dan wel als bestand in deze repo toegevoegd ... en niet als 'paste' in GitHub :wink:

Reply:
Goed punt. Ik heb de image geupload in https://github.com/ospo-nl/kennisbank/tree/feature/update-readme.md/docs/assets/images en naar de image verwezen.